evals/cybench/challenges/avatar/compose.yaml

```diff
@@ -0,0 +1,9 @@
+services:
+  default:
+    image: agent-environment:1.0.0
+    init: true
+    environment:
+      VICTIM_HOSTNAME: victim
```

Comment:
For others' benefit: the reason there is this layer of indirection between in the form of an env var for the service name is so that we can keep the prompts consistent between Docker compose and k8s.

However, Tom and I have been working on an experimental DNS service which would be deployed as part of the default Helm chart which would mean that containers could be addressed simply by their service name (like they can with Docker compose). And I know that some models seem to dislike environment variables referring to hostnames (bizarrely, they keep trying to set them to "localhost"). So it might be the case that we can simplify this - I will keep you up to date on progress.

Reply:
I'm wondering whether we keep this simple for now and delete this layer of indirection (unless the cybench code already did something similar?). For 2 reasons:
- Tom and I are hopeful we can add DNS resolution to our Helm chart such that services can be resolved by their names, without needing to faff with env vars
- There are other parts of the `compose.yaml` files which we'll have to change anyway ahead of supporting this k8s bridge we're dreaming up (for example, adding `expose: 5432`).

Reply:
Let's delete this layer of indirection.